php/api/controller/class-audit-posts-controller.php

```diff
@@ -1134,12 +1134,14 @@ public function get_altid_post( $request ) {
 		$post = new \WP_Query( $args );
 
 		if ( is_wp_error( $post ) || empty( $post ) || 0 === $post->post_count ) {
-			return new \WP_Error( 'rest_post_invalid_altid_lookup', __( 'Invalid post lookup.', 'tide-api' ), array(
+			$post = new \WP_Error( 'rest_post_invalid_altid_lookup', __( 'Invalid post lookup.', 'tide-api' ), array(
 				'status' => 404,
 			) );
+		} else {
+			$post = array_shift( $post->posts );
 		}
 
-		return array_shift( $post->posts );
+		return apply_filters( 'tide_api_modify_pre_response_post', $post, $request );
```

Comment:
@rheinardkorf Wouldn't this filter make more sense to be named `tide_api_get_altid_post`?

Reply:
makes sense